pilot/pkg/serviceregistry/kube/controller/controller.go

```diff
@@ -548,6 +572,13 @@ func (c *Controller) HasSynced() bool {
 func (c *Controller) SyncAll() error {
 	var err *multierror.Error
 
+	if c.nsInformer != nil {
+		ns := c.nsInformer.GetStore().List()
```

Comment:
Only get the system ns?

Reply:
Should effectively do the same thing no? Is there a perf benefit for looking up by name? If not I'd rather leave this.

Reply:
It has no diff in performance view, because only one namespace in local cache. 
Just cause confusion.